README.md

```diff
@@ -25,3 +25,11 @@ $ gem install jekyll
 ```bash
 $ jekyll serve
 ```
+
+A development server will run at `http://localhost:4000/`
+
+## Deployment
+
+Commits and merges into `master` will be deployed automatically to the production web server through webhook posts from GitHub.
```

Comment:
Do we want to talk a little bit more about setup? We need a place to lay out other config stuff (e.g., nginx) but maybe this isn't the right spot